.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [optional] kindly add your mit.md to your git ignore file, so it doesn't show remotely. Please, this is only optional since the previous reviewer did not mention it.

Reply:
Fixed.